101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+      "type": "bool",
+      "metadata": {
+        "description": "Value indicating whether the filter subjectBeginsWith and subjectEndsWith are case sensitive or not"
+      },
+      "defaultValue": false
+    }
+  },
+  "variables": {
+    "logicApp": {
+      "name": "[parameters('logicAppName')]",
+      "apiVersion": "2017-07-01",
+      "resourceId": "[resourceId('Microsoft.Logic/workflows', parameters('logicAppName'))]",
+      "location": "[parameters('location')]",
+      "triggerId": "[resourceId('Microsoft.Logic/workflows/triggers', parameters('logicAppName'), 'manual')]"
+    },
+    "eventGridTopic": {
+      "name": "[parameters('eventGridTopicName')]",
+      "apiVersion": "2018-05-01-preview",
+      "resourceId": "[resourceId('Microsoft.EventGrid/topics', parameters('eventGridTopicName'))]",
+      "location": "[parameters('location')]",
+      "schema": "[parameters('eventGridTopicSchema')]"
+    },
+    "eventGridSubscription": {
+      "name": "[parameters('eventGridSubscriptionName')]",
+      "apiVersion": "2018-05-01-preview",
+      "location": "[parameters('location')]",
+      "deliverySchema": "[parameters('eventGridSubscriptionDeliverySchema')]",
+      "endpoint": {
+        "type": "[parameters('eventGridSubscriptionEndpointType')]"
+      },
+      "filter": {
+        "includedEventTypes": "[split(parameters('eventGridSubscriptionIncludedEventTypes'), ',')]",
+        "isSubjectCaseSensitive": "[parameters('eventGridSubscriptionIsSubjectCaseSensitive')]"
+      }
+    },
+    "tags": {
+      "author": "Justin Yoo",
+      "profile": "https://twitter.com/justinchronicle",
+      "projectUrl": "https://github.com/Azure/azure-quickstart-templates",
+      "repositoryUrl": "https://github.com/Azure/azure-quickstart-templates",
+      "license": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/LICENSE"
+    }
+  },
+  "resources": [
+    {
+      "comments": "### RESOURCE - LOGIC APP - EVENT HANDLER ###",
+      "apiVersion": "[variables('logicApp').apiVersion]",
+      "type": "Microsoft.Logic/workflows",
+      "name": "[variables('logicApp').name]",
+      "location": "[variables('logicApp').location]",
+      "tags": "[variables('tags')]",
+      "properties": {
+        "state": "Enabled",
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {},
+          "triggers": {
+            "manual": {
+              "type": "Request",
+              "kind": "Http",
+              "inputs": {
+                "method": "POST",
+                "schema": {}
+              }
+            }
+          },
+          "actions": {},
+          "outputs": {}
+        }
+      }
+    },
+    {
+      "comments": "### RESOURCE - CUSTOM EVENT GRID TOPIC ###",
+      "apiVersion": "[variables('eventGridTopic').apiVersion]",
```

Comment:
Use a literal apiVersion so schema validation can be done.

Reply:
It passes validation.